core/constants.py

```diff
@@ -85,10 +88,17 @@ def get_package_file_contents(package: str, filepath: str) -> str:
         FileNotFoundError. The file does not exist.
     """
     try:
+        if mode == 'r':
+            with io.open(
+                os.path.join(package, filepath), mode, encoding='utf-8'
+            ) as file:
+                read_mode_data: str = file.read()
+                return read_mode_data
```

Comment:
Why is this needed?

Reply:
We need to read the image in the binary mode and for that we cannot specify the encoding because doing that will raise an error. Mainly this is required to read the images.